### ISSUE TITLE
[New] Approve Guideline for Disambiguations

### ISSUE THREAD

**Issue by AdultSun** [stashapp/StashDB-Docs#5]
The most glaring omission in the docs right now is the absence of any guidance for performer disambiguations. I punted on writing up an unofficial section at first because there didn't seem to be much of a consensus yet on how we wanted to format them, so it just says `TBD` [here](https://guidelines.stashdb.org/docs/performers/performer-names-and-aliases/#disambiguation-formatting).

We had a productive discussion started by Valmox with [this Discord thread](https://discord.com/channels/559159668438728723/1018134972957204561). Consensus there was leaning towards `YYYY, Studio` for disambiguations but it wasn't unanimous. There were a few competing suggestions in there as well. We may want to run an approval thread with multiple options to choose from instead of a typical Approve/Reject.

**Comment by Flashy78** [comment]
Not sure if either of the searches on StashDB or Stash rely on the order of a disambig. There's a chance having studio first might be more advantageous for searching, but I also may be making things up.

**Comment by Valmox** [comment]
I personally prefer the format as Flashy78 suggests. Even if it won't affect searching, if the alphabetical sort order was to consider disambiguation, then it will be better to go [Studio, YYYY] - this would group all "Jane" performers from Gammae together, rather than in date order. But, as AdultSun says, the consensus was petty much in favour of year first. I do think that the number of future changes the decision will result in could affect votes on this one.

**Comment by echo6ix** [comment]
I also agree with @Flashy78 and @Valmox wrt to a `Studio, YYYY` schema. When notating a source, whether it's a citation, bibliographic, or whatever, the standard is the title of source and the year after it.

But I offer my opinion with the usual caveat tangent that this kind of manual user entry should be eliminated altogether under the rule of thumb that less manual input = less data to moderate for quality = less user error on both input and voting. Disambiguation should be automated.

**Comment by AdultSun** [comment]
I updated this section during the big organization overhaul. It does not state a required order or format but explains the most common approaches with examples. We will still need to officially approve something for this section before recommending anything more specific than that.

I will also update this issue to reflect that we now have unofficial language and now need to approve and update it instead.

https://guidelines.stashdb.org/docs/performers/edit/disambiguation/disambiguation-formatting/
